google/ggrc-core
# GGRC-6632 Modify Control model to convert Option attributes to strings

# Issue description

Sync service send string value for Option attributes of Control. This value should be saved with Contol and displayed on UI. Fields that should be modified: kind, means, verify_frequency.

# Steps to test the changes

- Check if api requests work proper for Control (POST/PUT/GET/DELETE). Request and result body should contain kind, means, verify_frequency fields as strings.
- Check export of Control
- Check api that is based on revisions

# Solution description

Removed links between Control and Option models.
Added string fields "kind, means, verify_frequency" to Control.
Update old revisions in code to show option fields as strings.

# Sanity checklist


# Migration checklist


# PR Review checklist
